pom.xml

```diff
@@ -86,6 +77,35 @@
         </executions>
       </plugin>
     </plugins>
+    <pluginManagement>
+      <plugins>
+        <!-- Eclipse m2e lifecycle mapping: allow execution of 'os-maven-plugin plugin -->
+        <plugin>
+          <groupId>org.eclipse.m2e</groupId>
+          <artifactId>lifecycle-mapping</artifactId>
+          <version>1.0.0</version>
+          <configuration>
+            <lifecycleMappingMetadata>
+              <pluginExecutions>
+                <pluginExecution>
+                  <pluginExecutionFilter>
+                    <groupId>kr.motd.maven</groupId>
+                    <artifactId>os-maven-plugin</artifactId>
+                    <versionRange>[1.5.0.Final,)</versionRange>
```

Comment:
I believe fixed version is the best option. With concrete version for all deps/plugins you can have reproducible builds.